### ISSUE TITLE
Linked thumbnail images should be taken out of the flow for screenreaders

### ISSUE THREAD

**Issue by jbickar** [SU-SWS/stanford_person#20]
If you look at the View here (https://sites.stanford.edu/sociology/people/faculty), which is the stanford_person_faculty View, it generates accessibility errors because the images are hyperlinked and do not have descriptive alt text. Because we also are hyperlinking the neighboring h3 element to the same destination, best practice would be:

```
<a href="foo"><img src="img.jpg" /><h3>Jane Stanford</h3></a>
```

However, that might cause problems with styling. We may instead want to add the role="presentation" attribute to the img element.
